tests/tasks/cloud/test_flow_run.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from unittest.mock import MagicMock
+
+from prefect.tasks.cloud.flow_run import FlowRunTask
+
+
+@pytest.fixture()
+def client(monkeypatch):
+    cloud_client = MagicMock(
+        graphql=MagicMock(
+            return_value=MagicMock(
+                data=MagicMock(flow=[MagicMock(id="abc123"), MagicMock(id="def456")])
+            )
+        ),
+        create_flow_run=MagicMock(return_value="xyz890"),
+    )
+    monkeypatch.setattr(
+        "prefect.tasks.cloud.flow_run.Client", MagicMock(return_value=cloud_client),
+    )
+    yield cloud_client
+
+
+class TestFlowRunTask:
+    def test_initialization(self):
+        # verify that the task is initialized as expected
+        task = FlowRunTask(
+            project_name="Test Project",
+            flow_name="Test Flow",
+            parameters={"test": "ing"},
+        )
+        assert task.project_name == "Test Project"
+        assert task.flow_name == "Test Flow"
+        assert task.parameters == {"test": "ing"}
+
+    def test_flow_run_task(self, client):
+        # verify that create_flow_run was called
+        task = FlowRunTask(
+            project_name="Test Project",
+            flow_name="Test Flow",
+            parameters={"test": "ing"},
+        )
+        # verify that run returns the new flow run ID
+        assert task.run() == "xyz890"
+        # verify the GraphQL query was called with the correct arguments
+        client.graphql.assert_called_once_with(
+            {
+                "query": {
+                    'flow(where: { name: { _eq: "Test Flow" }, project: { name: { _eq: "Test Project" } } }, order_by: { version: desc })': {
```

Comment:
This test is failing in a lower version of python; I'm OK with this test but it also feels like one of those that could be frustrating to maintain if we add new outputs or anything.  Maybe we can relax what we test for?  Something like:
```python
arg = client.graphql.call_args_list[0][0]
assert "Test Flow" in arg
# etc.
```

just a thought, not a requirement

Reply:
Ah man, think I'm getting bitten by dict order issues. I'm fine with that proposal-- didn't see this issue locally.